network-api/networkapi/mozfest/templates/fragments/carousel/carousel_navigation.html

```diff
@@ -0,0 +1,25 @@
+<div class="swiper-navigation tw-relative tw-z-10 tw-mb-5 tw-hidden medium:tw-flex medium:tw-flex-row medium:tw-justify-between medium:items-center{% if not page.use_wide_template and page.signup  %} tw-ml-0 tw-pl-0 tw-container{% endif %}">
+  <h3 class="font-weight-bold">{{ title }}</h3>
+  <div class="tw-flex tw-flex-row tw-justify-between tw-items-center">
+    <div class="swiper-button-prev">
+      <svg class="swiper-button-icon" viewBox="0 0 21 21" fill="none">
```

Comment:
I imagine you've got a good reason for this, but just wondering if this could be an include?

Reply:
Ahh yes this was so we could change the currentColor on hover. Unfortunately there isn't a sprite sheet in this project.